app/views/profiles/search.html.erb

```diff
@@ -0,0 +1,98 @@
+		    	<tr>
+		    		<th class="header search-header">Summoner Search Result</th>
+		    	</tr>
+		    </thead>
+		    <tbody>
+		      <tr>
+		    			<td>
+		    			  <% if @friend %>
+			    		    <div id="friend-lookup-results" class="row well results-block">
+			    		      <div class="col-sm-5">
+			    		      	<strong>Summoner Name: <%= @name.capitalize %></strong><br>
+			    		        <strong>Summoner Level: <%= @friend.summoner_level %></strong><br>
+	                    <strong>Last Active: <%= @friend.revision_date.to_s.slice(0,10) %></strong><br>
+			    		      </div>
+			    		      <% if @statsObj %>
+			    		      	<div class="col-sm-5">
+			    		          <strong>Most Played Champ: <%= "#{@mpChamp[:name]}"%> </strong><%= " - (#{@mostPlayedChampObj.stats.total_sessions_played}x)"%> <br>
+			    		          <strong>Highest KDR Champ: <%= "#{@hkdrChamp[:name]}"%> </strong> 
+			    		                                    <% if @highestKillDeathRatioChampObj[0].stats.total_deaths_per_session >  0 %>
+	                                                  <%= " - (#{((@highestKillDeathRatioChampObj[0].stats.most_champion_kills_per_session.to_f)/(@highestKillDeathRatioChampObj[0].stats.total_deaths_per_session)).round(2)})"%>
+	                                                <%else%>
+	                                                  <%= " - (#{(@highestKillDeathRatioChampObj[0].stats.most_champion_kills_per_session_f.round(2))})"%>
+	                                                <%end%><br>
+	                      <strong>Highest WLR Champ: <%= "#{@hwlrChamp[:name]}"%> </strong> 
+			    		                                    <% if @highestWinLossRatioChampObj[0].stats.total_deaths_per_session >  0 %>
+	                                                  <%= " - (#{((@highestWinLossRatioChampObj[0].stats.most_champion_kills_per_session.to_f)/(@highestWinLossRatioChampObj[0].stats.total_deaths_per_session)).round(2)})"%>
+	                                                <%else%>
+	                                                  <%= " - (#{(@highestWinLossRatioChampObj[0].stats.most_champion_kills_per_session_f.round(2))})"%>
+	                                                <%end%><br>
+			    		        </div>
+			    		        <div class="col-sm-1">
+			    		        <!--While this displays a dynamic picture(most played champ), we might need to have users choose their preferred display pictures
+			    		          when searched for or on their profile. This could be done when editing profiles.-->
+				    		        <%= link_to image_tag("icons/#{@mpChamp[:name].delete(' ')}Square.png"), champion_path(@mpChamp.id)  %>
+				    		      </div>
+			    		      <% else %>
+			    		        <div class="col-sm-6 no-stats">
+			    		          <strong>No stats found for summoner.</strong><br>
+				    		        <strong>Either summoner is inactive or</strong><br>
+		                    <strong>there are no stats to display.</strong><br>
```

Comment:
This can be made into one line and you don't need the `<br>` elements since the text will wrap automatically inside the `<div>`.  
Change it to `<strong>No stats found for summoner. Either summoner is inactive or there are no stats to display.</strong>`